### ISSUE TITLE
Feature Request: Dark mode

### ISSUE THREAD

**Issue by GAndrews98** [robwhess/my-osu-website#2]
It would be nice to have a dark mode toggle if possible. Especially useful in dark classrooms...

**Comment by gnubufferoverflows** [comment]
Yes, definitely need to have dark mode. CS majors are mostly nocturnal creatures, and often easily scared when bright light infiltrates their natural habitat.
